src/components/Main.js

```diff
@@ -1,30 +1,56 @@
 // @flow
 import React from 'react';
+import { graphql } from 'react-apollo';
+import { compose, branch, renderComponent } from 'recompose';
+import gql from 'graphql-tag';
 
 import List, { type SubmissionInfo } from './List';
 import { Box } from './Layout';
 
-const fakeData: Array<SubmissionInfo> = [
-  {
-    title: 'The Criminal Underworld Is Dropping Bitcoin for Another Currency',
-    url:
-      'https://www.bloomberg.com/news/articles/2018-01-02/criminal-underworld-is-dropping-bitcoin-for-another-currency',
-    submitter: 'fragosti',
-    comments: [],
-  },
-  {
-    title: 'Bitcoin Is The MySpace Of The Cryptocurrency World (Part 1)',
-    url:
-      'https://seekingalpha.com/article/4135035-bitcoin-myspace-cryptocurrency-world-part-1',
-    submitter: 'rgmvisser',
-    comments: [],
-  },
-];
-
-const Main = () => (
+type Props = {|
+  submissions: Array<SubmissionInfo>,
+|};
+
+// Stateless component for loading state (to avoid if-statement within Main component)
+const MainLoading = () => <Box mx={2}> ...Loading </Box>;
```

Comment:
nit but maybe Loading... instead of ...Loading? Up to you

Reply:
Sounds good. Do you know where we should be keeping track of this stuff? Asana?

Reply:
Yeah, let's talk tonight, but maybe a bugs tag in asana with priority levels or something. idk. let's talk tonight